haystack/management/commands/update_index.py

```diff
@@ -248,6 +255,12 @@ def handle(self, **options):
         elif self.verbosity > 1:
             LOG.setLevel(logging.INFO)
 
+        if (minutes and age) or (minutes and start_date) or (age and start_date):
+            raise NotImplementedError("Minutes / age / start date options are mutually exclusive")
```

Comment:
This should call [`parser.error`](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.error) instead for consistency.

```suggestion
            parser.error("Minutes / age / start date options are mutually exclusive")
```

It could be further simplified by using [`ArgumentParser.add_mutually_exclusive_group()`](https://docs.python.org/3/library/argparse.html#mutual-exclusion) so the help will automatically show these as exclusive options.